sklearn/ensemble/bagging.py

```diff
@@ -387,6 +404,38 @@ def _validate_y(self, y):
         # Default implementation
         return column_or_1d(y, warn=True)
 
+    def _get_estimators_data_draws(self, sample_mask=False, feature_mask=False):
```

Comment:
I'm not convinced that you need those parameters for a private method used once in the code and once in testing.